program/shinyApp/R/enrichment_analysis/server.R

```diff
@@ -170,6 +170,13 @@ enrichment_analysis_geneset_server <- function(
 
           if(isTruthy(input$Notes) & !(isEmpty(input$Notes))){
             fun_LogIt(message = "<span style='color:#298c2f;'>**Personal Notes:**</span>")
```

Comment:
```suggestion
            fun_LogIt(message = "<span style='color:#298c2f;'>**Personal Notes:**</span>")
```
Can be removed I think as not Personal Notes in small and greens shows up but the 'Notes' are massiv